sailor/pai/alerts.py

```diff
@@ -0,0 +1,129 @@
+"""
```

Comment:
the module should be named "alert" (singular). we have this in assetcentral as well